docs/developer/decision-records/2022-04-21-dpf-blob-data-transfer/README.md

```diff
@@ -0,0 +1,38 @@
+# Blob storage transfer
+
+ADR describing the blob storage transfer end to end flow between 2 participants.
+
+## Description
+
+The data-plane-azure-storage extension can be used on DPF to support blob transfers.
+A client can trigger a blob transfer on the consumer side via the Data Management API.
+The client needs to use the managedResources=true option in its HTTP request, otherwise the transfer will not work. Client would not be aware that an error occurred, it would just never see the transferProcess with completed state when polling for the result.   
```

Comment:
under what situation this will happen? Was not clear to me from the doc.

`Client would not be aware that an error occurred, it would just never see the transferProcess with completed state when polling for the result.`

Reply:
If any failure happen, the client/consumer cannot know and will just be waiting for the transfer to happen. There is an [issue](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1242) open for it.